cmd/geth/chaincmd.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015 The go-ethereum Authors
+// Copyright 2013-2016 The go-ethereum Authors
```

Comment:
Please don't update the copyright years in source files. They should be the year when the file was added.